src/main/java/module-info.java

```diff
@@ -0,0 +1,15 @@
+module mssql.jdbc
```

Comment:
I know naming is hard, I just wanted to let you know there are other possible naming strategies and simply converting the artefact name to the module name in not necessarily the best option

https://github.com/jodastephen/jpms-module-names